Source/JavaScriptCore/runtime/IntlDurationFormat.cpp

```diff
@@ -330,8 +348,10 @@ static Vector<Element> collectElements(JSGlobalObject* globalObject, const IntlD
     auto scope = DECLARE_THROW_SCOPE(vm);
 
     bool done = false;
+    bool signDisplayed = true;
```

Comment:
It seems confusing to start out by saying that we already _did_ something; I think this would be clearer as `bool needsSignDisplay = false;`

Reply:
I used the variable names as they are in the specification, but I think your suggestion is clearer and better. Thank you.